lib/mix/lib/mix.ex

```diff
@@ -162,35 +162,56 @@ defmodule Mix do
   aliases defined in dependencies are not accessible from the current project.
 
   Aliases can be used very powerfully to also run Elixir scripts and
-  bash commands, for example:
+  shell commands, for example:
+
+      # priv/hello1.exs
+      IO.puts("Hello One")
 
-      # priv/hello.exs
-      IO.puts("hello")
+      # priv/hello2.exs
+      IO.puts("Hello Two")
 
       # priv/world.sh
       #!/bin/sh
       echo "world!"
 
       # mix.exs
-      defp create_aliases do
+      defp aliases do
         [
-          "taskalias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
-          "taskalias2": ["run priv/hello1.exs", "run priv/hello2.exs"]
+          alias1: ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
+          alias2: [
```

Comment:
I think we should only provide working code in the examples, unless it’s evident that the code is failing (like error/exceptions in doctests). I’ve seen so many cases in which developers just jump to the examples and skip the rest of the documentation. 

That’s why I would provide only `alias: [“run...”]`, keep an adapted version of the next paragraph and explain in the next one the “limitation”, something like:

> Mix tasks are designed to run only once. This prevents the same task to be executed multiple times. For example, if there are several tasks depending on `mix compile`, the code will be compiled once. Tasks can be executed again if they are explicitly reenabled using `Mix.Task.reenable/1`.
>
>     (here goes the example using Mix.Task.reenable/1)
>
> Some tasks are automatically reenabled: `mix cmd`...(here goes the list)
> (and maybe explain here that some tasks can receive several parameters, instead lf using the same tasks more than once)